include/gsl/span

```diff
@@ -258,6 +258,41 @@ namespace details
             return !(rhs > lhs);
         }
 
+#ifdef _MSC_VER
+        // MSVC++ iterator debugging support; allows STL algorithms in 15.8+
+        // to unwrap span_iterator to a pointer type after a range check in STL
+        // algorithm calls
+        friend constexpr void _Verify_range(span_iterator lhs,
+                                            span_iterator rhs) noexcept
+        {   // test that [lhs, rhs) forms a valid range inside an STL algorithm
+            Expects(lhs.span_ == rhs.span_ // range spans have to match
+                && lhs.index_ <= rhs.index_); // range must not be transposed
+        }
+
+        cosntexpr void _Verify_offset(const difference_type n) const noexcept
```

Comment:
`cosntexpr`?

Reply:
@annagrin fixed it when she merged. (I submitted a PR to fix it, but apparently certain people who shall remain nameless don't get Github notifications.)